Fix GH issue #5404 (DEV) (#5554)
corona-warn-app/cwa-app-android: .github/pull_request_template.md

### Checklist

__Thank you for this PR! Please consider the following:__

* To the _Community_ :heart::
    * Please link to an issue ticket where your code change has been greenlit, otherwise it's
      unlikely it can be merged.
    * Use a descriptive title: {task_name} (closes #{issue_number}),
      e.g.: `Use logger (closes # 41)`.
    * If this PR comes from a

>       fork,please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

Reviewer: ```suggestion
      fork, please [allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
```

    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*`
      and `/`
      or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for
      further information).
    * Before submitting a PR, please make sure that you target the active `release/*.*.*` branch
      which can be determined by the active milestone on GitHub

>     * Set `community` label

Author: Updated

>     * Include `community` in PR title such as "PR is fixing issue ### (community)"

Reviewer: ```suggestion
    * Include `COMMUNITY` in PR title such as "PR fixes issue ### (COMMUNITY)"
```
1. In the past mostly "COMMUNITY" was used, sometimes "Community" but never "community" (in lower case).
2. "is fixing" is a problematic grammar usage. "fixes" is timeless and better.

* To the _Maintainers_ :coffee::
    * Title structure: `Jira ticket name (EXPOSUREAPP-XXXX)`.
    * Set labels: `maintainers` and (`bug`or `text change`).
    * Update Jira status: `In Review`.
    * Fulfill internal `Acceptance Criteria`.
    * If mentioned in the Jira ticket link all corresponding GitHub issues in the sidebar.
* To _everyone_ :world_map::
    * Describe your changes in detail, if you changed the UI, screenshots or GIFs would be awesome!
    * Short step-by-step instructions help the reviewer test your changes, e.g. how to navigate to a
      new UI element you added.
    * The PR _won't be reviewed_ if CI is failing or if there are merge conflicts. If CI is still
      failing mark the PR as a draft and write a little comment on your status.
    * Provide at least a few unit and/or instrumentation tests.
    * Use a meaningful branch name. Use either `fix` or `feature` as prefix for your branch,
      e.g. `fix/prevent-npe-on-device-rotation-issue_123`

>     * Test your changes thoroughly. Only open PRs which you think is ready to be merged. If you

Reviewer: ```suggestion
    * Test your changes thoroughly. Only open PRs which you think are ready to be merged. If you
```
grammar
